flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlAlterTable.java

```diff
@@ -114,4 +110,8 @@ private void printIndent(SqlWriter writer) {
 	public String[] fullDatabaseName() {
 		return tableName.names.toArray(new String[0]);
 	}
+
+	public String[] fullNewDatabaseName() {
+		return newTableName.names.toArray(new String[0]);
```

Comment:
why would database name come from `newTableName.names.toArray(new String[0])`?

Reply:
Since now the newTableName can be written in `catlog`.`database`.`tableName` format, we should support get the full path.

Reply:
I see. The method name is quite confusing then. Rename to something link `fullTablePath()`?

Reply:
sorry，I misunderstand your meaning. I'll correct right.